packages/@dcl/legacy-ecs/src/decentraland/Components.ts

```diff
@@ -175,6 +176,38 @@ export class Transform extends ObservableComponent {
   }
 }
 
+/** @public */
+export enum AttachToAvatarAnchorPointId {
+  NameTag = 0,
```

Comment:
Should 0 (default) be the AvatarRoot instead?

Reply:
we talk about this with Alvaro and he decided not to add `AvatarRoot` as an option in this iteration, just the place where the name tag is and the hands